Solutions/Marain.Workflow.Api.Services.AspNetCore/Microsoft/Extensions/DependencyInjection/WorkflowEngineServiceCollectionExtensions.cs

```diff
@@ -95,6 +95,7 @@ public static class WorkflowEngineServiceCollectionExtensions
             services.AddTenantedWorkflowEngineFactory();
             services.AddTenantedAzureCosmosWorkflowStore();
             services.AddTenantedAzureCosmosWorkflowInstanceStore();
+            services.AddTenantedAzureCosmosWorkflowInstanceChangeLog();
```

Comment:
I think we should make the blob storage implementation the default here (I also think we should have a blob implementation of the workflow store which should be default, but that's a separate issue 🙂)

Reply:
I think we should open an issue for this as it has deployment implications too.

Reply:
Created #145 to track this.